### PR TITLE
added a new extra, add vertical lines

### PR DESCRIPTION
**Pull request by tylerjrichards**
@arnaudmiribel I made a quick new extra, but for some reason I'm having a bit of trouble following the contribution guidelines. how do new extras pop up in the gallery? Making this a draft pr for now

### CODE REVIEW

**Comment by blackary** [comment]
@tylerjrichards Because of some of the hacks that the gallery uses, you will probably need to stop and restart your streamlit app for it to show up, but after that it should show up automatically when you run the gallery locally

**Comment by tylerjrichards** [comment]
@blackary I made a new folder in src/streamlit-extras called add_vertical_space, then made a function inside it with the decorator, and added the dunderized documentation in the init file, and then ran streamlit run gallery/streamlit_app.py, but don't get anything, did I miss a step?

**Comment by blackary** [comment]
@tylerjrichards Nope, it works fine for me when I checked out your branch 
image

**Comment by tylerjrichards** [comment]
@blackary so weird that it isn't working for me, I updated streamlit-extras and streamlit and still it doesn't work locally. will take a look later

**Comment by tylerjrichards** [comment]
ooooook @blackary and I figured it out, and added to the contribution guide

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ Head over to our public [repository](https://github.com/arnaudmiribel/streamlit-
     __author__ = "Eva Jensen"
     __experimental_playground__ = False # Optional
     ```
+    To test it out, run `poetry run` and `pip install -e .` in the `streamlit-extras` directory, and then run the `gallery/streamlit_app.py` app.
```
Comment (blackary) [inline CONTRIBUTING.md:28]: `pip install -e .` alone should be sufficient

**Review by blackary — APPROVED** [review]
One tweak on contributor guide, but otherwise LGTM!

File: CONTRIBUTING.md (hunk 1 above)
Comment (tylerjrichards) [reply CONTRIBUTING.md]: great, removed poetry run!

**Pull request merged**